src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Account/layout.html.twig

```diff
@@ -12,5 +12,12 @@
 {% endblock %}
 
 {% block sidebar %}
-{{ knp_menu_render('sylius.frontend.account', {'template': 'SyliusWebBundle:Frontend:menu.html.twig'}) }}
+    {% if app.user %}
```

Comment:
I only want to check user is logged in, so not any special authorization needed, will `is_granted('IS_AUTHENTICATED_REMEMBERED')` cover all logged in cases!?